viewer/viewer-awt/src/main/java/org/icepdf/ri/common/views/annotations/FreeTextAnnotationComponent.java

```diff
@@ -127,9 +127,7 @@ public FreeTextAnnotationComponent(FreeTextAnnotation annotation, DocumentViewCo
                 KeyboardFocusManager.getCurrentKeyboardFocusManager();
         focusManager.addPropertyChangeListener(this);
 
-        if (!annotation.hasAppearanceStream()) {
-            resetAppearanceShapes();
-        }
+        resetAppearanceShapes();

```

Comment:
Rebuilding a FreeText annotation is fairly complicated and sometimes doesn't look as good as the original.  Does the fix rely on this change?  If it does there might be something else wrong with the generated content stream.

Reply:
Yes, it's basically the whole fix. I've noticed that as soon as you move (or change any property of) the annotation, it appears, and given that mouseDragged called resetAppearanceShapes, I simply removed the condition.     
Like you said, I see now that it's not a good fix, and there's probably a problem with the stream.

Reply:
Any chance you have time to put a test procedure together with a sample file that I can take a look at?  I half to admit that 'if' block has come and gone over the years.

Reply:
GitHub can't upload the file for some reason, so [here](https://www.dropbox.com/s/twbmd49l0it93g9/OneFreeText.pdf?dl=0) is a Dropbox link. I could also send it by mail if you prefer.     
It's simply a blank pdf file with a FreeText annotation created with IcePdf in the middle of the page.

Reply:
I grabbed a copy of the file made quite a few edits to the file using a build off master.  I didn't have trouble seeing the edits on open.  Anything I'm missing in the  test case?

Reply:
That's strange. I just sent you a mail with a recording of the problem.

Reply:
Apparently I need a better monitor.  I my dev box  I've been resizing the page so that I can see the whole page.  If I don't resize and just scroll down I don't see the annotation.  I'm thinking this is likely a validate/repaint issue.  Will keep looking.